components/card/card.css

```diff
@@ -0,0 +1,186 @@
+.card-description {
+    font-style: italic;
+    font-weight: 600;
+}
+
+.card-flex {
+    flex-wrap: wrap;
+    gap: 1rem;
+}
+
+.card-horizontal{
+    flex-flow: column wrap;
+    gap: 1rem;
+    min-width: 70vw;
+    height: 15rem;
+    padding: 1rem;
+}
+
+.card {
+    background-color: var(--white);
+    overflow: hidden;
+    object-fit: fill;
+    width: 25rem;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+    
+}
+
+.card-shadow {
+    box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.image{
+    position: relative;
+}
+
+.image img {
+    height: 15rem;
+    width: auto;
+    background-repeat: no-repeat;
+}
+
+.image-horizontal img {
+    height: 20rem;
+    width: 15rem;
+    background-repeat: no-repeat;
```

Comment:
Suggestion: Using fixes width and height could create problems for responsiveness, you can use max-width and max-height property.